phase1/tf.jsonnet

```diff
@@ -98,7 +98,7 @@
           ),
         },
       },
-      resource: {
+      resource+: {
```

Comment:
I assume this is what removes the need for `std.mergePatch`? What is this called in jsonnet?